datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -102,7 +103,7 @@
   protected final boolean _pausePartitionOnError;
   protected final Duration _pauseErrorPartitionDuration;
   protected final long _processingDelayLogThresholdMillis;
-  protected final boolean _enablePollDurationMillisMetric;
+  protected final boolean _enableAdditionalHistogramMetrics;
```

Comment:
can you make this name generic and avoid using Histogram

Reply:
done